### PR TITLE
Release 0.4.7

### PR DESCRIPTION
**Pull request by alvinhui**
- feat: Store 间相互调用
- docs: Store 内的 Action 说明

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
...
+const icestore = new Icestore();
+const stores = icestore.registerStores({
+  task,
+  user,
+});
+
+export default stores;
+```
+
+#### Pay attention to circular calling
```
Comment (temper357) [inline README.md]: 这里使用 circular dependencies 循环依赖的表述感觉意思更清晰点

File: README.md [hunk 2]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
...
+
+Please pay attention to circular calling when call each other between Stores.
```
Comment (temper357) [inline README.md]: Please pay attention to circular dependencies problem when methods calling each other between stores.

Comment (temper357) [inline README.md]: Please pay attention to circular dependencies problem when methods calling each other between stores.

File: README.md [hunk 3]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
...
+});
+
+export default stores;
+```
+
+#### Pay attention to circular calling
+
+Please pay attention to circular calling when call each other between Stores. 
+
+For example, the a method in Store A calls the b method in Store B, and the b method in Store B calls the a method in Store A, which will form a dead cycle.
```
Comment (temper357) [inline README.md]: For example, the A method in Store A calls the B method in Store B and the B method in Store B calls the A method in Store A will results into an endless loop.

File: README.md [hunk 4]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
...
+
+If multiple stores call each other, the occurrence probability of the dead cycle problem will increase.
```
Comment (temper357) [inline README.md]: Be careful the possibility of endless loop problem will arise when methods from different stores call each other.

File: README.md [hunk 5]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
 * Return value
   - void
 
-## Advanced use
+## Advanced Use
+
+### Chain react of State
```
Comment (temper357) [inline README.md]: store 联动 个人感觉用 store interaction 更好点，或者看看有没有更地道的表述

File: README.md [hunk 6]
```diff
@@ -205,9 +210,75 @@ otherwise apply middleware the store by namespace.
...
+
+The chain react is a common requirement, you can implement it by calling other Store in Store.
```
Comment (temper357) [inline README.md]: Store interaction is  a common usage scene which can be implemented by calling actions from other  store in a store's action.

File: README.md (hunk 3 above)
Comment (temper357) [reply README.md]: method 是从对象层面描述的，action 是从 store 成员的职能角度描述，感觉使用 action 描述更好些

**Comment by temper357** [comment]
0.4.x 增加了几个新功能，CodeSandbox 中的示例建议也同步更新下

**Comment by temper357** [comment]
Reviewed.

File: README.md (hunk 1 above)
Comment (alvinhui) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (alvinhui) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (alvinhui) [reply README.md]: done

File: README.md (hunk 4 above)
Comment (alvinhui) [reply README.md]: done

File: README.md (hunk 5 above)
Comment (alvinhui) [reply README.md]: done

File: README.md (hunk 6 above)
Comment (alvinhui) [reply README.md]: done

**Comment by alvinhui** [comment]
英文文档已更新根据 Review 意见

**Pull request merged**